skbeam/core/accumulators/binned_statistic.py

```diff
@@ -620,7 +620,7 @@ def __call__(self, values, statistic=None):
             The values of the selected statistic in each bin.
         """
         # check for what I believe could be a common error
-        if values.shape != self.expected_shape:
+        if tuple(values.shape) != tuple(self.expected_shape):
```

Comment:
yeah basically I'm running into a case where I saved shape as a tuple in a database and when I retrieve it it's returned as a list (I used sqlite, although perhaps I made an error somewhere, this is from memory)
However, maybe it's good to still raise since it helped to hint an sqlite problem, so maybe better to leave as is?